css/main.css

```diff
@@ -335,6 +335,7 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
```

Comment:
- [ ] You have done very well with adding the `Required` attribute and `Maxlength="500"` on the `<textarea>` but you must increase the `width` and reduce the `height` to align with the `name` and `email` input tags. Refer to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) for clarity on this.

![review (3)](https://user-images.githubusercontent.com/71140133/197336878-0fa4b8b6-14eb-4f59-9c59-3d07db127d75.PNG)